release/release_inputs.json

```
@@ -1,5 +1,7 @@
 {
   "release_cromwell.githubToken": "<<GithubAPIToken>>",
   "release_cromwell.organization": "<<broadinstitute for a real release, your github username if testing>>",
-  "release_cromwell.majorRelease": <<true for a major release, false for a minor release>>
+  "release_cromwell.majorRelease": <<true for a major release, false for a minor release>>,
+  "release_cromwell.publishHomebrew": true,
+  "release_cromwell.checkout_as_ssh": false
```

```suggestion
  "release_cromwell.checkout_as_ssh": true
```

Alternatively could be a `<<>>` doc line, like the others above.